ensemble-view/src/main/java/hec/ensembleview/StatComputationHelper.java

```diff
@@ -126,21 +163,36 @@ private static float[] computeStatFromMaxAvgDurationComputable(EnsembleTimeSerie
         MetricCollectionTimeSeries mct = ets.iterateAcrossEnsembleTracesWithSingleComputable(
                 new MaxAvgDuration(value));
 
-        return mct.getMetricCollection(selectedZdt).getComputedValuesForStatistic(stat);
+        return mct.getMetricCollection(selectedZdt).getComputedValuesForStatistic();
     }
 
     private static float[] computeStatFromMaxAccumDurationComputable(EnsembleTimeSeries ets, Statistics stat, ZonedDateTime selectedZdt, int value) {
         MetricCollectionTimeSeries mct = ets.iterateAcrossEnsembleTracesWithSingleComputable(
                 new MaxAccumDuration(value));
 
-        return mct.getMetricCollection(selectedZdt).getComputedValuesForStatistic(stat);
+        return mct.getMetricCollection(selectedZdt).getComputedValuesForStatistic();
     }
 
     private static float[] computeStatFromTotalComputable(EnsembleTimeSeries ets, Statistics stat, ZonedDateTime selectedZdt) {
         MetricCollectionTimeSeries mct = ets.iterateAcrossEnsembleTracesWithSingleComputable(
                 new Total());
 
-        return mct.getMetricCollection(selectedZdt).getComputedValuesForStatistic(stat);
+        return mct.getMetricCollection(selectedZdt).getComputedValuesForStatistic();
+    }
+
+    private static float[] computeStatFromNDayComputable(EnsembleTimeSeries ets, Statistics stat, ZonedDateTime selectedZdt, float[] value) {
```

Comment:
After your refactor, the "Statistics" parameter for this method and several above it is no longer used and can be removed from the method signature.

Reply:
Concur. Removed the Statistics as a parameter